spec/models/solr_document_spec.rb

```diff
@@ -75,4 +75,36 @@
     end
   end
 
+  describe 'SolrDocument concerns' do
```

Comment:
We test the other concerns here: https://github.com/sul-dlss/argo/tree/develop/spec/models/concerns

Could we keep the same pattern? Or move those specs here?

I think the expectation, is that I should be able to find the respective specs in a similar location.